.github/workflows/metrics.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches:
       - master
 
+  repository_dispatch:
+    types: [dispatch-metrics]
+
+  workflow_dispatch:
```

Comment:
does `workflow_dispatch` work with no value?

Reply:
Yup! This is apparently legal YAML https://github.com/opensciencegrid/docker-software-base/blob/master/.github/workflows/build-container.yml#L11

See the nifty `Run workflow` button that it adds here: https://github.com/opensciencegrid/docker-software-base/actions/workflows/build-container.yml

Reply:
So much for me believing that yaml was a respectable language  ;)